input/resources/au-core-medicationrequest.xml

```diff
@@ -247,9 +247,9 @@
       <constraint>
         <key value="au-core-medreq-03"/>
         <severity value="error"/>
-        <human value="Date shall be at least to day"/>
-        <expression value="toString().length() &gt;= 10"/>
-        <xpath value="f:matches(effectiveDateTime,/\d{4}-[01]\d-[0-3]\dT[0-2]\d:[0-5]\d([+-][0-2]\d:[0-5]\d|Z)/)"/>
+        <human value="Date is precise to the day or, if not available, the Data Absent Reason extension shall be present"/>
```

Comment:
I think we still want the shall in the first part of the human expression:
- Date shall be precise to the day or, if not available, the Data Absent Reason extension shall be present

apply to all instances